Shared/Controller/NotificationSettingsController.swift

```diff
@@ -53,6 +62,57 @@ class NotificationSettingsController: ObservableObject {
         self.$showMilestoneAdded.removeDuplicates().sink { newValue in
             if(!self.refreshing) { UserDefaults.shared.showMilestoneAddedNotification = newValue }
         }.store(in: &self.subscribers)
+        
+        
+        self.$customAmountInput.sink(receiveValue: {newValue in
+            if let doubleValue = Double(newValue) {
+                //no currency formatting fallback (e.g. 300000)
+                UserDefaults.shared.set(doubleValue, forKey: "customNotificationAmount")
+                self.rejectedInputShowing = false
+            } else {
+                //currency formatter (e.g. $300,000)
+                let f = NumberFormatter()
+                f.numberStyle = .currency
+                if let number = f.number(from: newValue) {
+                    DispatchQueue.main.async {
+                        self.rejectedInputShowing = false
+                        UserDefaults.shared.set(number.doubleValue, forKey: "customNotificationAmount")
+                        print(UserDefaults.shared.double(forKey: "customNotificationAmount"))
```

Comment:
I'd have a look in Loggers.swift and user an existing logger or make a new one?

Reply:
i added this for debugging, i forgot to remove it